.travis.yml

```diff
@@ -16,7 +16,7 @@ services:
   - docker
 
 before_install:
-  - git clone https://github.com/certbot/certbot
+  - git clone --depth=1 -b no-keyauthorization https://github.com/certbot/certbot
```

Comment:
Can you file a Pebble issue to link with a TODO here to revert to Certbot master when they've fixed this upstream?